Modules/Devices/src/Devices.Application/ApplicationOptions.cs

```diff
@@ -11,6 +11,11 @@ public class ApplicationOptions
 
     [Required]
     public PaginationOptions Pagination { get; set; } = new();
+
+    [Required]
+    [MinLength(3)]
+    [MaxLength(45)]
```

Comment:
Why 45? With this we will never reach the max characters of 100 - or do we?

Reply:
We do! We can go with 48 now that we removed the realm, leaving a margin of two chars (or we can go with 50 with no margin).